### PR TITLE
Add Example Java project, update doc, add carrier codes

### PR DESCRIPTION
**Pull request by joostoo**

### CODE REVIEW

File: example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        .setDestination("ORD")
+        .setDepartureDate(1302797146231L)
+        .setArrivalDate(1302797154231L)
+        .setOutgoing(false)
+        .setFareCode("VYXAP2")
+        .setCabinClass("V")
+        .setAvailableSeats(3)
+        .build();
+
+    // Create a fare for the one-way segment(s)
```
Comment (plaflamme) [inline example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java]: Why does it say "one-way"?

File: example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+   *  This is the code that gets called every time you want to publish a message to the queue
+   *  @throws IOException
+   */
+  public synchronized void send(TripBatch tripBatch) throws IOException {
+    URL url = new URL("https://api.hopper.com:9998/xxx");
+    HttpURLConnection urlc = (HttpURLConnection) url.openConnection();
+    urlc.setDoOutput(true);
+    urlc.setRequestMethod("POST");
+    urlc.setRequestProperty("X-Point-Of-Sale", "US");
+    urlc.setRequestProperty("Content-Type", "application/x-protobuf");
```
Comment (plaflamme) [inline example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java]: I'm not sure if `HttpUrlConnection` handles `Content-Length` properly, but it should probably be set.

Comment (joostoo) [inline example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java]: Added urlc.setFixedLengthStreamingMode(tripBatch.getSerializedSize()); below

File: example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```
Comment (jder) [inline example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java]: Can we put a pretty ASCII hopper bunny here?

Comment (muerl) [inline example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java]: (+1)

Comment (bejar37) [inline example/src/main/java/com/hopper/rabbit/example/HopperFeedClient.java]: +100

**Pull request merged**